local-tests/setup/shiva-client.ts

```diff
@@ -291,3 +270,23 @@ export class ShivaClient {
     }
   }
 }
+
+async function _processTestnetResponse<T>(
+  response: Response
+): Promise<TestNetResponse<T>> {
+  let createTestnet: TestNetResponse<T>;
+  try {
+    createTestnet = (await response.json()) as TestNetResponse<T>;
+  } catch (err) {
+    let message = await response.text();
+    console.error('Error while creating testnet instance:', message);
```

Comment:
hmmm -- this isn't always used for creating a testnet instance anymore; could be due to a `stopTestnet` or a `transitionEpoch` call.  Since we're `throw`ing from this function in this catch() anyway, maybe just :hocho: the `console.error()` entirely?